requirements.txt

```diff
@@ -42,11 +44,13 @@ ipfshttpclient==0.6.1
 itsdangerous==2.0.0a1
 jinja2==3.0.0a1
 jsonschema==3.2.0
+libusb1==1.8
```

Comment:
Can this requirement be problematic?

Reply:
Potentially - on CI for example.  I've taken a few preliminary steps to prevent it from being imported.  This dependency will certainly change the requirements a bit which can be reflected in the docs.  Perhaps we can also consider using a pip extra for this situation as we did before? `pip install nucypher[llama]`